### PR TITLE
NP-46711 Add NPI to Chapter

### PR DESCRIPTION
**Pull request by joachimjorgensen**
# Description

Link to Jira issue: [NP-46711](https://unit.atlassian.net/browse/NP-46711)

Legger til NPI fagfelt i registreringswizard for Kapittel, likt som for bok.

# Checklist

Ensure that the changes are aligned with the expectations of PO/designer before marking the PR as ready for review.

Also ensure that the following criterias are met:

- [x] The changes are working as expected
- [x] The changes are tested OK for different screen sizes
- [x] The changes are tested OK for a11y
- [x] Interactive elements have data-testids
- [x] I have done a QA of my own changes

_Note: some of these criterias may not always be relevant, and can simply be marked as completed._

[NP-46711]: https://unit.atlassian.net/browse/NP-46711?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/registration/resource_type_tab/sub_type_forms/ChapterForm.tsx
Comment (elkebab) [inline src/pages/registration/resource_type_tab/sub_type_forms/ChapterForm.tsx:1]: Må sikkert legges til visning av dette på landing page også?

Comment (joachimjorgensen) [reply src/pages/registration/resource_type_tab/sub_type_forms/ChapterForm.tsx:1]: Ser ut som det allerede er?

image

Comment (elkebab) [reply src/pages/registration/resource_type_tab/sub_type_forms/ChapterForm.tsx:1]: Det der er vitenskapelig kapittel da, når du legger til feltet i ChapterForm mistenker jeg du også må legge til visningen for kapittel i landing page

**Review by elkebab — APPROVED** [review]
Ikke testa, men koden ser bra ut!

**Pull request merged**